.prettierrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "overrides": [
+    {
+      "files": ["*.ts", "*.js"],
+      "options": {
+        "arrowParens": "avoid",
+        "printWidth": 120,
+        "semi": false,
+        "tabWidth": 2,
```

Comment:
I didn't know about the popularity of 2 spaces until I read up on some of the literature on it. I guess it makes sense, since JavaScript code does tend to have a lot more nesting than others. I don't have a strong preference for either, so 2 spaces SGTM

Reply:
Yeah, I personally much prefer 2 spaces due to all of the nesting in JS. We can always change this down the line, and in theory prettier should be able to automagically fix any issues :)